adafruit_ntp.py

```diff
@@ -53,9 +57,8 @@ def __init__(
         :param int socket_timeout: UDP socket timeout, in seconds.
         """
         self._pool = socketpool
-        self._server = server
-        self._port = port
-        self._packet = bytearray(48)
+        self._socket_address = self._pool.getaddrinfo(server, port)[0][4]
```

Comment:
@anecdata good call out on the `init`. I'll move this to the `datetime` property:
```
if self._socket_address is None:
   self._socket_address = self._pool.getaddrinfo(server, port)[0][4]
```
Will hold on other reviews first